assets/src/adaptivity/operators/contains.ts

```diff
@@ -68,7 +68,9 @@ export const containsAnyOfOperator = (inputValue: any, conditionValue: any) => {
     if (!isNaN(inputValue)) {
       return conditionArray.includes(parseFloat(inputValue));
     } else {
-      return conditionArray.some((v) => inputValue.toString().includes(v));
+      return conditionArray.some((v: any) =>
```

Comment:
When comparing `fact notContainsAnyOf "hazard,hazzard,blink,flash,indicator,emergency,help,assistance"`, it should not be case sensitive and both lower case or upper case answer should be accepted. 

We already handled this for `EqualTo `operator.